android/assets/jsons/translations/German.properties

```diff
@@ -404,7 +404,7 @@ An enemy [unit] has captured [cityName] = Eine feindliche Einheit [unit] hat [ci
 An enemy [unit] has captured our [ourUnit] = Eine feindliche Einheit [unit] hat unsere Einheit [ourUnit] gefangen genommen
 An enemy [unit] has destroyed our [ourUnit] = Eine feindliche Einheit [unit] hat unsere Einheit [ourUnit] zerstört
  # Requires translation!
-Your [ourUnit] has destroyed an enemy [unit] = 
+Your [ourUnit] has destroyed an enemy [unit] = Dein [ourUnit] hat einen gegnerischen [unit] zerstört
```

Comment:
Ich muß das endlich mal lernen, was wann in die templates gehört und was nicht... Und welche dort abgelegten Kommentare es in die Einzelsprachen "schaffen". Dieser Text wird momentan nur für die Zitadelle benutzt - wenn man zu Fuß angreift sieht man ja was passiert ist. Nicht so wenn beim Nächste-Runde-berechnen die Zitadellen ihre Punkte verteilen. Zeigt auch daß das ganze Übersetzungssystem mit Geschlecht nicht kann. Wofür mir keine Lösung einfällt...

Reply:
Alles was in "template.properties" ist, landet in den anderen Sprach-Dateien zum übersetzen. Hast du die Zeile manuell in das Template eingefügt?
Wie kann denn eine Zitadelle eine Einheit zerstören? Ich dachte, die gibt nur zusätzlichen Verteidigungsbonus für die Einheit, die auf dem Feld steht.
Im Englischen gibts eben keine Unterschiede bei dem Artikel zu den Geschlechtern. Deutsch ist da halt komplizierter und wir werden damit leben. :-)

Reply:
Nein Zitadellen folgen den Regeln wie auf einem [gewissen wiki](https://civilization.fandom.com/wiki/Citadel_(Civ5)) nachzulesen recht gut. -30 Punkte für jede Einheit die mindestens 1 feindliche Zitadelle in den 7 Nachbarn hat, in MapUnit.kt Funktion getCitadelDamage()... Den "unique" dazu haben wir bestimmt übersetzt, muß die Civilopedia anzeigen. Ich hatte nur die Idee, daß die Notizen oben rechts mehr hergeben könnten als vorher. Und Text den code auswirft und der nicht aus den json's stammt muß man zu Fuß in die template tun, ja.